lib/iris/fileformats/netcdf.py

```diff
@@ -2244,7 +2244,10 @@ def is_valid_packspec(p):
                        shuffle, fletcher32, contiguous, chunksizes, endian,
                        least_significant_digit, packing=packspec)
 
-        conventions = CF_CONVENTIONS_VERSION
+        if iris.config.netcdf.conventions_override:
+            conventions = cube.attributes['Conventions']
```

Comment:
@dkillick The `cube.attributes` may not necessarily have any `Conventions` to use ...